docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+Fork and clone the repository,create a branch for your whatever you wanna work on, then add, commit and push to your forked repository and create a PR. Do ensure there is no conflict in your PR. Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
+
+Fixes # (issue)
+
+## Type of change
```

Comment:
Why do you have `##` here? You do know it's gonna make this title smaller right?

Reply:
Okay, I will remove it now